pkg/services/serviceaccounts/api/api.go

```diff
@@ -147,7 +147,7 @@ func (api *ServiceAccountsAPI) getAccessControlMetadata(c *models.ReqContext, sa
 		return map[string]accesscontrol.Metadata{}
 	}
 
-	return accesscontrol.GetResourcesMetadata(c.Req.Context(), permissions, "serviceaccounts", saIDs)
+	return accesscontrol.GetResourcesMetadata(c.Req.Context(), permissions, "serviceaccounts", "id", saIDs)
```

Comment:
nit: could use `accesscontrol.ScopeAttributeID` instead of `id` here.

Reply:
I changed the implementation a bit, instead of using resource and resourceAttribute we take a prefix and validated scopes to that instead